ISSUE_TEMPLATE.md

```diff
@@ -1,18 +1,31 @@
-<!--
-  Thanks for your interest in using the App Center SDK for React Native.
+### **Description**
 
-  If you have an issue in React Native platform but have not integrated
-  App Center SDK in your application, please contact support by clicking
-  on the blue icon on the lower right of App Center portal (https://appcenter.ms).
-  Our support team will get in touch to assist you.
+Please describe the issue you are facing or a feature you would like to be added to the SDK.
 
-  Any issues related to SDK, please provide App Center SDK version and technical information
-  about your environment. So please run the following two commands and attach 
-  the output to your ticket:
+<!-- If making a feature request, remove the below information -->
+### **Repro Steps**
 
-  `react-native info`
-  `pod --version`
+Please list the steps used to reproduce your issue.
 
-  For bugs, please describe the reproduction scenario and expected behavior.
-  Please share sample code or any relevant file to help reproducing the issue.
--->
+1.
+2.
+
+### **Details**
+
+1. Which SDK version are you using?
+    - e.g. 1.5.0
+2. Which OS version did you experience the issue on?
+    - e.g. Android 8.1.0, iOS 11
+3. What device version did you see this error on?  Were you using an emulator or a physical device?
+    - e.g. Google Pixel 2 emulator, iPhone X physical device
+4. What third party libraries are you using?
+    - example
+5. Run the following command and paste the output below: `react-native info`
+
+Place output here
+
+6. Run the following command and paste the output below: `pod --version`
+
+Place output here
+
+7. Please enable verbose logging for your app using `AppCenter.setLogLevel(Log.VERBOSE)` before your call to `AppCenter.start(...)` and include the logs here:
```

Comment:
This instruction is not the right for react native, please rephrase the instruction from https://docs.microsoft.com/en-us/appcenter/sdk/other-apis/react-native#adjust-the-log-level:

> In iOS, call [MSAppCenter setLogLevel: MSLogLevelVerbose]; before any call to [AppCenterReactNative register]; (or AppCenterReactNativeAnalytics or AppCenterReactNativeCrashes or AppCenterReactNativePush) in AppDelegate.m. You have to add @import AppCenter; if missing in that file. 
> In Android, call AppCenter.setLogLevel(android.util.Log.VERBOSE); before SoLoader.init in onCreate callback in MainApplication.java.